setools/policyrep/terule.pxi

```diff
@@ -476,11 +476,13 @@ cdef class FileNameTERule(BaseTERule):
     cdef:
         Type dft
         readonly str filename
+        uint32_t match_type
```

Comment:
This won't be visible in Python code, only in cython code. It needs the `readonly` qualifier. By making it visible, I think it should change to a `PolicyEnum` object.

Reply:
Hi, sorry for a late reply.

I have changed the type for a new `PolicyEnum` subclass that I have created and updated usages of it.